lib/logger.js

```diff
@@ -11,14 +10,9 @@ function Logger () {
     return new Logger()
   }
 
-  try {
-    // From everywhere in the application require.main.filename is the original node script path.
-    const moduleConfigFile = path.resolve(path.dirname(require.main.filename), 'module.yml')
-    const module = Yaml.load(moduleConfigFile)
-    this.moduleName = module.name
-  } catch (error) {
-    throw new Error('Logger couldn\'t find module.yml in root directory of node application.')
-  }
+  // As a rule of thumb, launcher would run the process in
+  // the module directory which is in the name of the module
+  this.moduleName = path.basename(path.resolve())
```

Comment:
Could you explain why the logger now decides the module name by the folder?

Reply:
I think we should talk about it, May be the solution is that the logger will get the module name from outside..
@idanstark42 Would appreciate your comment

Reply:
MS requires the module directory to be the same as written in the modules.yml. We can add a environment variable to the launcher.

I agree this might be weird option, but getting the module name from outside feel to me even weirder.

Reply:
I thought the MS determined the location and structure of the module.yml file. Anyways, it seems weird to me to take it from the folder instead of the module.yml because that seems like the point of the 'name' field on the module.yml, but if it's really less error prone maybe it's better this way. Anyways because the MS is very strict it didn't seem to mind too much. How come it causes errors in the clock?

Reply:
The point of the module.yaml is to be used by the launcher not by the module. The thing is that I don't like to use IO on critical things like the logger.

Reply:
I thought the Module.yml is for every part of the infastructure to use. Also, Reading the folder name is still IO...
But again it doesn't matter to me too much, It seems more or less the same. If @taltaub22 is OK with this I think we can merge this

Reply:
Im ok with that, just wanted another opinion because it changes everywhere